Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -574,8 +575,11 @@ LoadEventNexus::runLoadNexusLogs(const std::string &nexusfilename, T localWorksp
     if (run.hasProperty("period_log")) {
       auto *temp = run.getProperty("period_log");
       // Check for corrupted period logs
+      std::string status = "";
       std::unique_ptr<TimeSeriesProperty<int>> tempPeriodLog(dynamic_cast<TimeSeriesProperty<int> *>(temp->clone()));
-      checkForCorruptedPeriods(std::move(tempPeriodLog), periodLog, nPeriods, nexusfilename);
+      nPeriods = checkForCorruptedPeriods(std::move(tempPeriodLog), periodLog, nPeriods, nexusfilename, status);
```

Comment:
Is `nPeriods` used anywhere after being returned?

Reply:
`nPeriods` is an output from the parent method, it's used [here](https://github.com/mantidproject/mantid/blob/7680bac7757ce51ac42e7fd3fb6b2ad8829f1036/Framework/DataHandling/src/LoadEventNexus.cpp#L918C42-L918C42).

Reply:
ah I see thanks